service/templates/display_title.html

```diff
@@ -108,7 +108,8 @@ <h3 class="heading-medium">Owner(s)</h3>
 
                                     {% for proprietor in title.proprietors %}
                                         <!-- Proprietor, multiple addresses -->
-                                        <h4 class="heading-small expand-bottom-half">{{ proprietor.name }}</h4>
+                                        <h4 class="heading-small expand-bottom-half">{{ proprietor.name }}{{proprietor.name_supplimentary}}{{proprietor.name_information}}{{proprietor.charity_name}}{{proprietor.trading_name}}{{proprietor.country_incorporation}}</h4>
```

Comment:
Thought so too, but in reality the proprietor would only have 1 or 2 of these things maximum, just a potential to have all of them...